doc/api/os.md

```diff
@@ -509,6 +509,48 @@ On POSIX systems, the operating system release is determined by calling
 available, `GetVersionExW()` will be used. See
 <https://en.wikipedia.org/wiki/Uname#Examples> for more information.
 
+## `os.guessHandleType(handle)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `handle` {integer} The handle number to try and guess the type of.
+
+* Returns: {string}
+
+Returns the type of the handle passed in, or `'INVALID'` if the provided handle
+is invalid.
+
+Currently, the following types for a handle can be returned:
+
+<table>
+  <tr>
+    <th>Constant</th>
+  </tr>
+  <tr>
+    <td><code>TCP</code></td>
+  </tr>
+  <tr>
+    <td><code>TTY</code></td>
+  </tr>
+  <tr>
+    <td><code>UDP</code></td>
+  </tr>
+  <tr>
+    <td><code>FILE</code></td>
+  </tr>
+  <tr>
+    <td><code>PIPE</code></td>
+  </tr>
+  <tr>
+    <td><code>UNKNOWN</code></td>
+  </tr>
+  <tr>
+    <td><code>INVALID</code></td>
+  </tr>
+</table>
```

Comment:
Why not use markdown table or list? That would be much more readable

Reply:
I honestly debated adding a secondary column describing the meaning of each of the values, but then I realized that the values are pretty self-explanatory. Will turn to a normal markdown list 👍

Reply:
Pushed, not sure if this is what you were referring to (also not sure if I should wrap the values in quotes or not)